### DISCUSSION TITLE
Has anyone recreated powerlevel10k prompt for starship?

### DISCUSSION THREAD

**Discussion by darelover** [starship/starship#1967] [Q&A]
I use powerlevel10k with following preferences:
```
Prompt Style: Rainbow
Character Set: Unicode
Show current time?: No
Prompt Separators: Angled
Prompt Heads: Sharp
Prompt Tails: Flat
Prompt Height: One Line
Prompt Spacing: Compact
Icons: Many Icons
Prompt Flow: Concise
Enable Transient Prompt?: No
Instant Prompt Mode: Quiet
```
I would like to recreate this prompt for starship and was wondering if anyone has already done this?

**Comment by carlosplanchon** [comment]
I wanted exactly this (rainbow style, one line, compact, flat heads and flat tails),
could not find it, so I built it and it is now a proper preset:

https://github.com/carlosplanchon/starship-p10k-rainbow

![p10k above, the preset below](https://raw.githubusercontent.com/carlosplanchon/starship-p10k-rainbow/main/screenshots/hero.png)

Same segment order and ANSI colors as p10k's default rainbow profile:
the yellow dirty-state subsegment that collapses when the repo is clean, detached
HEAD with tag, and REBASE 3/10 style progress. One `curl | sh` installs
it (with a timestamped backup of your current config). And since Starship
is cross-shell, the look survives even if you leave zsh :)
